src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -248,20 +249,37 @@ export class ManagerDashboardComponent implements OnInit, OnDestroy {
     }
   }
 
+  confirmResetPin() {
+    const confirmationMessage = $localize`:@@managerDashboard.resetPinConfirm:Resetting the PIN may temporarily disrupt connections for devices using this server until they reconnect. Do you want to continue?`;
```

Comment:
Let's add the confirmation message to `dialogs-prompt component` to stick to our standard pattern and avoid adding the `extraMessage`